import-service/handlers/importProductsFile/importProductsFile.ts

```diff
@@ -0,0 +1,52 @@
+import { APIGatewayProxyEventBase, APIGatewayEventDefaultAuthorizerContext } from 'aws-lambda';
+import 'source-map-support/register';
+import AWS from 'aws-sdk';
+import createError from 'http-errors';
+import middy from '@middy/core'; 
+import httpErrorHandler from '@middy/http-error-handler';
+import cors from '@middy/http-cors';
+import httpSecurityHeaders from '@middy/http-security-headers';
+import { BUCKET } from '../../config';
+
+const importProductsFile = middy(async (event: APIGatewayProxyEventBase<APIGatewayEventDefaultAuthorizerContext>) => {
+  console.log( {event} );
+
+  const { name } = event.queryStringParameters;

```

Comment:
It's better to check if `queryStringParameters` exists at all because if we go to this link https://8lcxyejzqd.execute-api.eu-west-1.amazonaws.com/dev/import and don't enter any query params, you lambda will fail with 500 error, because `queryStringParameters` will be undefined in this case.

Reply:
thanks, I'll fix it.